### DISCUSSION TITLE
Chrome ignoring CSS font-weight property

### DISCUSSION THREAD

**Discussion by Swamy571** [cefsharp/CefSharp#3387] [General]
I am using font-family 'Proxima Nova'. Chrome seems to be using `font-weight: lighter`; as default?, 

I am using 'font-weight: normal' and 'color: black' for a grid, but text in the column still looks like not full black color. It showing lighter color, but I want full black color.

@font-face {
  font-family: 'ProximaNova-Regular';
  src: url("fonts/ProximaNova-Regular.eot?#iefix") format("embedded-opentype"), url("fonts/ProximaNova-Regular.woff") format("woff"), url("fonts/ProximaNova-Regular.ttf") format("truetype");
  font-weight: normal;
  font-style: normal;
}

.k-grid{
    color: black !important;
    border-width: 0px;
    font-family: 'ProximaNova-Regular' !important;
  }

![image](https://user-images.githubusercontent.com/78414794/106593952-71d17c80-6577-11eb-9b9c-830172bd0511.png)

**Comment by wnunezc** [comment]
i think you have the blurry problem: 

Cef.EnableHighDPISupport();

**Reply by Swamy571** [reply]
Still, it is showing same.

**Reply by wnunezc** [reply]
you can show your code how you have implemented cef ...

**Comment by amaitland** [comment]
How does it behave when you load the web page directly in Chrome? If it works in Chrome then you'll probably need a newer version.

**Comment by Swamy571** [comment]
Hi Guys,
My sincere apologies, previously I used font face of 'Proxima Nova.ttf' file instead of 'Proxima Nova.otf'. So now I used 'Proxima Nova.otf' file and working as expected with brighter color.

Best Regards,
Swamy.
